pkg/placement/membership.go

```diff
@@ -325,15 +329,17 @@ func (p *Service) performTableDissemination(ctx context.Context) error {
 	p.disseminateLock.Lock()
 	defer p.disseminateLock.Unlock()
 
-	state := p.raftNode.FSM().PlacementState()
+	state := p.raftNode.FSM().PlacementState(false)
+	stateWithVirtualNodes := p.raftNode.FSM().PlacementState(true)
```

Comment:
I wonder if we can use `p.minAPILevel` to check if there are any older daprds connected to the service. If not, we don't need to calculate `stateWithVirtualNodes`. 
I think that's what that property is, but I have to double-check. If anyone knows and confirm that for me in the meantime, it would b great.

Reply:
`minAPILevel` is cluster-wide. Although in the state we do keep the API level of each connected host.

Reply:
Great! Thank you! So I'll add a check, if `minApiLevel` is >=20, I won't calculate`stateWithVirtualNodes`. 🙏

Reply:
Since we're on topic:

> We attach the api level of the daprd as metadata to the stream connection. The placement service will look at this data and decide if it needs to send the vnodes based on it (needed for daprd sidecars on older versions). This way we're not relying on the minimum API level of the cluster, but serve data based on what the particular sidecar needs.

Just a note that if a sidecar tries to connect to Placement and the reported API level is lower than the cluster's, it's rejected (at least, rejected as actor host).

Reply:
Thanks for pointing that out. I went at looked at https://github.com/dapr/dapr/pull/7134 in the meantime so it's more clear now.
I guess the comment still applies though, right? It would be a case where placement is upgraded before the daprd-s. I can update the comment to reflect this better.

Reply:
Yes I think your code is fine. My comment was more of a clarification on how this works.